CONTRIBUTING.md

```diff
@@ -7,6 +7,101 @@
   * [Suggesting Enhancements](#suggesting-enhancements)
   * [Pull Requests](#pull-requests)
 
+:+1::tada: First off, thanks for taking the time to contribute!
+
+You can look at the issues [with help wanted label][help-wanted] for items that
+you can work on.
+
+If you need help, please feel free to reach out to our [slack channel](https://app.slack.com/client/T09NY5SBT/C019LQYGC5C)!
```

Comment:
This link is not correct. It should point to kubernetes workspace

![Screenshot from 2021-01-07 17-27-03](https://user-images.githubusercontent.com/6303309/103890168-9ae42600-510d-11eb-83a9-090e47216962.png)

Reply:
If the person already logged into Kubernetes slack. That link will open the channel.